src/features/cell-renderers/FormattedUser.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
```

Comment:
This needs to return a string instead of a component for virtualize table to auto add the string to a title for tooltip.